samples/cubes/convertStatusToString.js

```diff
@@ -1,96 +1,78 @@
+let EnumDSSCubeStates = require('./EnumDSSCubeStates');
+
 module.exports = (status) => {
-  const EnumDSSCubeStates = {
-    DssCubeReserved : 0,
-    DssCubeProcessing : 1,
-    DssCubeActive : 2,
-    DssCubePersisted : 4,
-    DssCubeInfoDirty : 8,
-    DssCubeDirty : 16,
-    DssCubeLoaded : 32,
-    DssCubeReady : 64,
-    DssCubeLoadPending : 128,
-    DssCubeUnloadPending : 256,
-    DssCubePendingForEngine : 512,
-    DssCubeImported : 1024,
-    DssCubeForeign : 2048,
-    DssCubeRemapDirty : 4096,
-    DssCubeUpgrade : 8192,
-    DssCubeEmma : 16384,
-    DssCubeFromMSTRFile : 32768,
-    DssCubeHidden : 65536
-  }
   let results = {};
```

Comment:
Since you're not replacing "results" completely, but rather changing a child of this object, this can be `const`. `let` is only when you plan on completely editing the pointer (the root object). 

Changing something like a string or number variable requires let, but changing the contents of an array or object doesn't require `let` unless you completely overwrite the array or object with a completely new array or object.